New endpoints

ga-hegsvold opened the pull request.

ga-hegsvold committed:
Support for more endpoints & cleaned up endpoint names

- addressesaddresscategories
- addressescounties
- addressescountries
- addressescountriespostalcodes
- addressesmunicipalities
- arrangements
- debtors
- membershipsgroupsmemberships
- organizations
- organizationsboardmembers
- organizationsboards
- organizationsboardsboardfunctions
- organizationscategories
- personsmemberships
- subscriptionsdefinitions
- subscriptionsdefinitionssubscriptions
- access token put in new secret 'membercare-api-key'

ga-hegsvold committed:
Change system token secret back to 'api_key'

BaardBouvet commented on templates/subscriptionsdefinitionssubscriptions.json:
```diff
@@ -0,0 +1,40 @@
+          ],
+          [
+            "add",
+            "::properties",
```
Use `"namespaced_identfiers": false` in all the connector pipes instead of `"add_namespaces": false`, that way you avoid adding `::`. We should add a validation check for this in sesam-py to catch this going forward.

ga-hegsvold replied:
Fixed.

ga-hegsvold committed:
Use namespaced_identifiers:false instead of prefixing with '::'

ga-hegsvold committed:
Remove tracing & deprecated template 'counties'

ga-hegsvold committed:
Add missing collect pipes & reuse datatype templates

ga-hegsvold committed:
Remove deprecated datatype templates

ga-hegsvold committed:
Remove redundant property add_namespaces:false

ga-hegsvold committed:
Replace add_namespaces with namespaced_identifiers

- All connector pipes must have namespaced_identifiers:false
- Remove redundant template addressesmunicipalities
